src/utils/cluster_setup.py

```diff
@@ -0,0 +1,40 @@
+import click
+import subprocess
+import os
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.pass_context
+def cluster_setup(ctx: click.Context):
+    # Copy current environment
+    env = os.environ.copy()
+    commands = [
+        [
+            "oc",
+            "apply",
+            "--server-side",
+            "-k",
+            "https://github.com/kubeflow/trainer.git/manifests/overlays/runtimes?ref=master",
+        ],
+        [
+            "oc",
+            "apply",
+            "--server-side",
+            "-k",
+            "https://github.com/kubeflow/trainer.git/manifests/overlays/runtimes?ref=master",
+        ],
+        [
+            "oc",
+            "apply",
+            "--server-side",
+            "-k",
+            "github.com/kubeflow/training-operator.git/manifests/overlays/standalone?ref=v1.8.1",
```

Comment:
we can upgrade to 1.9.1 so it's inline with the version included in latest RHOAI version.